aten/src/ATen/native/cuda/Unique.cu

```diff
@@ -36,7 +36,7 @@ std::tuple<Tensor, Tensor, int64_t> compute_unique(
 
   // inverse indices
   Tensor inverse_indices;
-  if (!return_inverse) {
+  if (!return_inverse || num_inp == 0) {
```

Comment:
This fixes support for zero-sized tensors

Reply:
Hm what was the error? I'd think that in the else block inverse_indices would be allocated as zero-sized tensor, which should be fine.

Reply:
In the else block, there is a
```C++
Tensor inv_loc = at::empty({num_inp}, options);  // zero sized tensor here
.....
inv_loc[0] = 0;  // RuntimeError: index 0 is out of bound for size 0
```
so I would completely skip the else block, but we still need to initialize `inverse_indices` as a zero sized tensor